docs/config/README.md

```diff
@@ -27,7 +27,15 @@ You can define and run a pipeline by combining these three types of various buil
 
 The list of build-in modules can be found on [Modules Page](module/README.md).
 
-Examples of configuration files are listed in the [Examples Page](../../examples/README.md), so try to find and arrange a configuration file that is close to the data processing you want to perform.
+Examples of configuration files are listed in the [Examples Page](../../examples/README.md), so try to find and arrange a configuration file that is close to the data processing you want to perform. 
+
+When we want to use streaming pipeline, we need to enable a settings option for streaming. [Reference](https://github.com/kouzoh/mercari-segmentation-jp/discussions/365#discussioncomment-528168)
+```JSON
+"settings": {
+    "streaming": true
+  }
+```
+A example of streaming pipeline with pubsub as source and sink can be found [here](https://github.com/codechaitu/DataflowTemplate/blob/master/examples/streaming-pubsub-to-beamsql-to-pubsub.hjson).
```

Comment:
The link will be modified if the corresponding PR is ok. 
[Reference](https://github.com/mercari/DataflowTemplate/pull/22)